### PR TITLE
Women pagev1

### PR DESCRIPTION
**Pull request by KytaihoraVolodymyr**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -15,10 +15,25 @@ GEM
...
+    nokogiri (1.11.3-x64-mingw32)
+      racc (~> 1.4)
     nokogiri (1.11.3-x86_64-darwin)
+=======
+    mini_mime (1.1.1)
+    mini_portile2 (2.6.1)
+    nokogiri (1.12.3)
+      mini_portile2 (~> 2.6.1)
+>>>>>>> upstream/master
+      racc (~> 1.4)
```
Comment (AndriiMish) [inline Gemfile.lock]: this is not resolved conflict results
======= and >>>>>>> upstream/master shouldn't be here

File: pages/women_page.rb [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    PAGE_TITLE = 'WOMEN'
+
+    def page_title
+        #TBD to move to parent page
+        browser.find_element(class: 'cat-name')
+    end
+
+#TITLES OF CATEGORIES
+    def categories_filter
+        browser.find_element(xpath: "//span[contains(@class,'layered_subtitle') and text()='Categories']")
```
Comment (AndriiMish) [inline pages/women_page.rb:11]: You seem to already have updated version with moving locating element to separate method

Comment (KytaihoraVolodymyr) [reply pages/women_page.rb:11]: Done

File: Gemfile.lock (hunk 1 above)
Comment (KytaihoraVolodymyr) [reply Gemfile.lock]: Done

**Comment by AndriiMish** [comment]
Addressed in other PR

**Pull request closed without merging**